src/allegro_flare/drawing_interface_svg.cpp

```diff
@@ -27,7 +27,8 @@ namespace allegro_flare
    {}
 
 
-
+    // Has to be defined even if pure virtual, see https://stackoverflow.com/a/15113814
```

Comment:
Can you remove this comment?  I think there were a few other places pure virtuals are defined and the extra comments aren't needed.

Reply:
Well, this case is very special. It is needed ONLY because we are building a shared lib.